bin/commands/scripts/build.js

```diff
@@ -1,30 +1,30 @@
+const { join } = require('path');
 const runSeries = require('run-series');
-const runParallel = require('run-parallel');
 const { spawn } = require('child_process');
-const { babel } = require('../../util/executables');
 const noop = require('../../util/noop');
+const copyTemplate = require('../../util/copy-template');
+const { MODULE_FORMATS, NODE_ENVIRONMENTS } = require('../../../lib');
 
-module.exports = ({ callback = noop, flags }) => {
-    const cjsOptions = ['src', '--out-dir', 'lib', '--copy-files', ...flags];
-    const esmOptions = ['src', '--out-dir', 'es', '--copy-files', ...flags];
+const distTemplateDirectory = join(__dirname, '../../../templates/dist');
+const distTargetDirectory = join(process.cwd(), 'dist');
+
+const build = ({ env, format, flags, cb }) =>
+    spawn('npx', ['rollup@2', '-c', ...flags], {
+        stdio: 'inherit',
+        env: {
+            ...process.env,
+            NODE_ENV: env,
+            MODULE_FORMAT: format,
+        },
+    }).on('close', cb);
 
-    const runner = flags.indexOf('--watch') >= 0 ? runParallel : runSeries;
-    runner(
-        [
-            // Build CommonJS modules
-            cb => {
-                spawn(babel, cjsOptions, {
-                    stdio: 'inherit',
-                    env: { ...process.env, BABEL_ENV: 'cjs' },
-                }).on('close', cb);
-            },
-            // Build ES modules (default)
-            cb => {
-                spawn(babel, esmOptions, {
-                    stdio: 'inherit',
-                }).on('close', cb);
-            },
-        ],
-        callback
-    );
+module.exports = ({ callback = noop, flags }) => {
+    const steps = [
+        cb => copyTemplate(distTemplateDirectory, distTargetDirectory, {}, cb),
+        cb => build({ env: NODE_ENVIRONMENTS.DEV, format: MODULE_FORMATS.CJS, flags, cb }),
```

Comment:
I still need to wrap this in a check for watch flag. That way we only run the dev esm build.